docker/Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN pip install --no-cache-dir --user --upgrade ibm-generative-ai[lm-eval]
 RUN pip install --no-cache-dir --user -r server/requirements.txt
 # Clone the Git repository and install the Python package
 RUN git clone https://github.com/EleutherAI/lm-evaluation-harness.git && \
-    cd lm-evaluation-harness && \
+    cd lm-evaluation-harness &&  git checkout 568af943e315100af3f00937bfd6947844769ab8 && \
```

Comment:
NIT: Assuming this will need to be removed once merged?

Reply:
This is related to the issue that the latest version of lm-evaluation-harness isn't compatible with the bam module. The problem is the new `--trust_remote_code` arg in the lm-evaluation-harness. I will see if I can update the bam module here: https://github.com/IBM/ibm-generative-ai/blob/main/src/genai/extensions/lm_eval/model.py later